features/bootstrap/FeatureContext.php

```diff
@@ -624,4 +662,147 @@ private function composer_command($cmd) {
 		$this->proc( $this->variables['COMPOSER_PATH'] . ' ' . $cmd )->run_check();
 	}
 
+	/**
+	 * Initialize run time logging.
+	 */
+	private static function log_run_times_before_suite( $event ) {
+		self::$suite_start_time = microtime( true );
+
+		$travis = getenv( 'TRAVIS' );
+
+		// Default output settings.
+		self::$output_to = $travis ? 'stdout' : 'error_log';
```

Comment:
Yes, `STDERR` doesn't make much sense - I'll get rid of it and make `STDOUT` the default always, keeping the `error_log` option (my preference).